packages/block-library/src/embed/util.js

```diff
@@ -114,14 +119,24 @@ export const createUpgradedEmbedBlock = (
 		} );
 	}
 
+	const wpVariation = getBlockVariations( DEFAULT_EMBED_BLOCK )?.find(
+		( { name } ) => name === 'wordpress'
```

Comment:
Some comment as above to retrieve hardcoded value. Since `name` and `providerNameSlug` are the same now, choose what fits more. For example if you choose only the providerName, change this `find` check to use the same property.

Reply:
TBH for the sake of future refactoring, I'd rather stick with `name` here, since it'll be one thing less to think about for future devs who'd like to phase out `providerNameSlug`. (Semantically, it also makes more sense to say that we're looking for the right block variation by its name, rather than its provider name.)